secretflow/ml/nn/fl/fl_model.py

```diff
@@ -619,18 +623,22 @@ def save_model(self, model_path: Union[str, Dict[PYU, str]], is_test=False):
         res = []
         for device, worker in self._workers.items():
             assert device in model_path, f'Should provide a path for device {device}.'
-            if not os.path.exists(model_path[device]):
-                os.makedirs(model_path[device])
+            device_model_path, device_model_name = model_path[device].rsplit("/", 1)
```

Comment:
For input '/a/b' and '/a/b/', we got different output path '/a'，'/a/b'. But '/a/b' and '/a/b/' could mean the same dir.

Reply:
model_path must be a/b/c， a/b/c/will be banned